model/schema/expression.yaml

```diff
@@ -511,6 +513,18 @@ slots:
     range: GOTerm
     multivalued: false
 
+  cellular_component_ribbon_term:
+    description: >-
+      The primary cellular component slim term referred to within the context of an AnatomicalSite object and used by the expression ribbon.
+    domain: AnatomicalSite
```

Comment:
@adamgibs shouldn't this be a GOTerm not an AnatomicalSite?

Reply:
`domain: AnatomicalSite` is correct but it does need `range: GOTerm` too

Reply:
Okay, I've just updated it.